test/functional/plugin/lsp/completion_spec.lua

```diff
@@ -168,6 +168,71 @@ describe('vim.lsp.completion: item conversion', function()
     eq(expected, result)
   end)
 
+  it('uses exists highlight group for kind', function()
+    local completion_list = {
```

Comment:
is this list meant to be exhaustive (i.e. does it need to cover ALL highlight group examples), or could it be reduced to cover various kinds of edge cases